students/christopher_gantt/lesson07/html_render.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+class Element(object):
+    tag = "html"
+    indent = "    "
+
+    def __init__(self, content=None, **kwargs):
+        if content is None:
+            self.contents = []
+        else:
+            self.contents = [content]
+        self.attributes = kwargs
+
+
+    def append(self, new_content):
```

Comment:
class methods should only have one line space (module functions have two lines space)